include/ffconf.h

```diff
@@ -227,8 +227,11 @@
 /  Instead of private sector buffer eliminated from the file object, common sector
 /  buffer in the file system object (FATFS) is used for the file data transfer. */
 
-
-#define _FS_EXFAT	0
+#if defined(CONFIG_FS_FATFS_EXFAT)
+#define _FS_EXFAT	CONFIG_FS_FATFS_EXFAT
+#else
+#define _FS_EXFAT   0
+#endif

```

Comment:
Wouldn't it be enough to just  `_FS_EXFAT` the value of `CONFIG_FS_FATFS_EXFAT`?

Reply:
OK my bad. It is common file.

Reply:
There could be a chance that user doesn't defined CONFIG_FS_FATFS_EXFAT, so we might need to remain _FS_EXFAT in default setting as 0.